src/machines.scss

```diff
@@ -102,3 +102,8 @@
 #storage-pool-delete-modal span.pf-v5-c-check__body {
     margin-block-start: 0;
 }
+
+.vm-description {
+    white-space: pre-wrap;
+    overflow-wrap: break-word;
+}
```

Comment:
We could also have a `hyphens: auto;` in there; it's relatively new-ish in baseline 2023 and will hyphenate things it needs to wrap with break-word.

https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens

> The hyphens CSS property specifies how words should be hyphenated when text wraps across multiple lines. It can prevent hyphenation entirely, hyphenate at manually-specified points within the text, or let the browser automatically insert hyphens where appropriate.